new-cdc/eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/status/CDCStatus.java

```diff
@@ -0,0 +1,5 @@
+package io.eventuate.local.common.status;
+
+public enum CDCStatus {
+  CONNECTING, STARTED, PUBLISHING
```

Comment:
What's the difference between CONNECTING and STARTED?

Reply:
Connecting is initial state.
Started state says that cdc service started and listen for changes.

Reply:
I would have two states: STARTING and STARTED.
PUBLISHING is indicated by the counter > 0

Reply:
Ok, will do